static/css/headref_display.css

```diff
@@ -0,0 +1,136 @@
+html {
+    height: 100%;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+}
+
+body {
+    height: 100%;
+    background-color: #222;
+}
+
+.container {
+    padding-top: 2vw;
```

Comment:
should this have `box-sizing: border-box;` so that the padding doesn't make it larger than 100%? not sure if this matters for your layout